aiohttp/connector.py

```diff
@@ -964,6 +980,15 @@ async def _wrap_create_connection(
             async with ceil_timeout(
                 timeout.sock_connect, ceil_threshold=timeout.ceil_threshold
             ):
+                if self._network_interface:
+                    *args, sock_hostname, sock_port = args  # type: ignore[assignment]
```

Comment:
Honestly, I'd prefer having a new named variable rather than some generic `args` that is not clear what it contains at different points.


@Dreamsorcerer WDYT?

Reply:
See the 2nd part of my previous comment: https://github.com/aio-libs/aiohttp/pull/7132#discussion_r1052622495

If you agree with my analysis, the we can just replace `*args` from the parameter list and have 3 positional arguments instead.

Reply:
So which one is it?

Reply:
@Dreamsorcerer if we're confident that it's always 3 args, enumerating the m explicitly seems like a great idea.

Reply:
Well, it's definitely only used in one place in our code. I'd only be concerned if a user subclasses this and then passes extra arguments or something. But, it's not documented anywhere or anything, so I'd assume people are not doing that.

So, I'd say it's safe to convert to fixed arguments.